pkg/monitor/clustermonitor.go

```diff
@@ -53,13 +53,17 @@ func GetClusterClaimInfo(managedCluster *clusterv1.ManagedCluster) (string, int6
 		if claimInfo.Name == "id.openshift.io" {
 			clusterID = claimInfo.Value
 		}
+		if clusterID == "" && claimInfo.Name == "id.k8s.io" {
+			clusterID = claimInfo.Value
+		}
```

Comment:
@willkutler We use the clusterID to get the data from console.redhat.com - That ID should be a openshift ID . Why do we want this block ?

Reply:
@xrajesh this block will only get hit if the cluster is not an openshift cluster. If the cluster isn't an openshift cluster then `id.openshift.io` will be blank, and since `ClusterNeedsCCX` uses `clusterID` as a key, we need to set that value to something else. If the cluster isn't openshift, the clusterID won't be used for anything else.